### ISSUE TITLE
feat: allow setting sticker dimension

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1474]
To reduce the sizing of stickers, you can currently lower the framerate and duration, but not output resolution (currently 512x512). People should be able to set the output resolution (as a square) to reduce output webp sizes, specifically for mp4 conversions.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.9.5](https://github.com/open-wa/wa-automate-nodejs/compare/3.9.4...3.9.5) (2021-03-14)

- Release 3.9.5 [`385bb8d`](https://github.com/open-wa/wa-automate-nodejs/commit/385bb8d785bbe5558e6ffec7ae633ad4913ec621)
- ✨  added `square` in `Mp4StickerConversionProcessOptions` to set sticker output dimention #1474 [`a55ee43`](https://github.com/open-wa/wa-automate-nodejs/commit/a55ee43903611d5e72b92bff73fd9a51c0cb60e8)
- 🔧  patch logging improvements + redundancy link #1473 [`fe557f1`](https://github.com/open-wa/wa-automate-nodejs/commit/fe557f1e23252a00251a5419e46cb7520f80c78a)
- 🐛 fix trailing slash issue in sticker endpoints [`3c179cf`](https://github.com/open-wa/wa-automate-nodejs/commit/3c179cfbe8fd387e8f3e26c31de6165b3771d631)
- ✏️  typo fix [`00130b1`](https://github.com/open-wa/wa-automate-nodejs/commit/00130b1a8dfeb6c08829323cd57d36da488e363c)
